pkg/apis/flowcontrol/install/install.go

```diff
@@ -38,5 +39,10 @@ func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(flowcontrolv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(flowcontrolv1beta1.AddToScheme(scheme))
 	utilruntime.Must(flowcontrolv1beta2.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(flowcontrolv1beta1.SchemeGroupVersion, flowcontrolv1beta2.SchemeGroupVersion, flowcontrolv1alpha1.SchemeGroupVersion))
+	utilruntime.Must(flowcontrolv1beta3.AddToScheme(scheme))
+	// TODO: This controls serialization order, for 1.26, we can set the
```

Comment:
Since there is a TODO added, would you mind to create an issue and get it tracked? Thank you

Reply:
sure, please see https://github.com/kubernetes/kubernetes/issues/112512

Reply:
thanks, can you put that issue number in the TODO comment to close the loop?

Reply:
Yes, done.